fastlane/Fastfile

```diff
@@ -911,8 +934,38 @@ platform :ios do
   end
 
   desc 'Run localization only'
```

Comment:
Nit: Worth making this `desc` a bit more descriptive? 🤷 

```suggestion
  desc 'Updates the main `Localizable.strings` file — that will be imported by GlotPress — from code and `Info.plist` files'
```

Reply:
Might as well while we're at it 👍